test/test_sparse_csr.py

```diff
@@ -1080,6 +1086,30 @@ def _test(t, x, y):
                 y = self.genSparseCSRTensor((dk, dj), nnz1, device=device, dtype=dtype, index_dtype=index_dtype)
                 _test(t, x, y)
 
+                # Test mm(CSR, CSR)
+                x = self.genSparseCSRTensor((di, dk), nnz0, device=device, dtype=dtype, index_dtype=index_dtype)
+                y = self.genSparseCSRTensor((dk, di), nnz0, device=device, dtype=dtype, index_dtype=index_dtype)
+                _test_mm(x, y)
+
+                # Test mm(CSR, CSC)
+                x = self.genSparseCSRTensor((di, dk), nnz0, device=device, dtype=dtype, index_dtype=index_dtype)
+                y = self.genSparseCSRTensor((di, dk), nnz0, device=device, dtype=dtype, index_dtype=index_dtype)
+                y = y.transpose(0, 1)
```

Comment:
Nit:
```suggestion
                y = self.genSparseCSCTensor((dk, di), nnz0, device=device, dtype=dtype, index_dtype=index_dtype)
```
to ensure that `y` is and always will be a CSC tensor.